### DISCUSSION TITLE
[Detector Support]: motion detection is a lot less sensitive compare to person occupancy

### DISCUSSION THREAD

**Discussion by nguyentrangia** [blakeblackshear/frigate#10861] [Detector Support]
### Describe the problem you are having

![Screenshot_2024-04-06-11-30-45-596_io homeassistant companion android-edit](https://github.com/blakeblackshear/frigate/assets/65263385/1fa5763a-623e-4968-b2c7-5b55e262930c)
As the screenshot, occupancy makes a lot more sense than motion detection. What can be done to make motion detection better? All configs left default.

### Version

1.13.2

### Frigate config file

```yaml
NA
```

### docker-compose file or Docker CLI command

```yaml
NA
```

### Relevant log output

```shell
NA
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

USB

**Comment by NickM-27** [comment]
Motion detection has a maximum off time of 30 seconds meaning there has to be no motion for 30 seconds for it to get switched off. This is configurable
